src/main/java/org/skypro/skyshop/hwspringboot/controller/ShopController.java

```diff
@@ -0,0 +1,37 @@
+package org.skypro.skyshop.hwspringboot.controller;
+
+import org.skypro.skyshop.hwspringboot.marketService.SearchService;
+import org.skypro.skyshop.hwspringboot.marketService.StorageService;
+import org.skypro.skyshop.hwspringboot.model.article.Article;
+import org.skypro.skyshop.hwspringboot.model.product.Product;
+import org.skypro.skyshop.hwspringboot.searchEngine.SearchResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Collection;
+import java.util.List;
+
+@RestController
+@RequestMapping("/myservice")
```

Comment:
базового пути быть не должно

```suggestion
@RequestMapping
```